lib/wombat/build.rb

```diff
@@ -95,6 +95,19 @@ def prepare_azure()
         resource_group = Azure::ARM::Resources::Models::ResourceGroup.new
         resource_group.location = wombat['azure']['location']
 
+        # Create hash to be used as tags on the resource group
+        tags = {
+          InUse: "true"
+        }
+
+        # If an owner has been sepcified in the wombat file then add this as a tag
+        if wombat.key?('owner') && !wombat['owner'].nil?
+          tags[:owner] = wombat['owner']
```

Comment:
one should expect people will share their wombat.yml and no one will change the owner tag in the process